dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -390,6 +391,11 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 									collection,
 									front.isNetworkFront,
 								)}
+								leftContent={
+									collection.displayName === 'Opinion' && (
```

Comment:
There seems to be [a slightly different mechanism in frontend to define this](https://github.com/guardian/frontend/blob/49425b9a1d392e211a6b78eba40f25e4f19bf35b/common/app/layout/FaciaContainer.scala#L90-L101).

I wonder if we should keep it the same so production don't get unexpected results?

Interesting comment there too to not hard code this, but not sure we have other examples to justify it.